### PR TITLE
Contact Form 

### PR DESCRIPTION
**Pull request by sainingo**
# Created Contact Form
> The form sends data to me email

- There's form validation handling using html5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -203,11 +203,11 @@ <h4>About Me</h4>
...
              if you'd like to get chat please get in touch</h4>
-             <form>
-                 <input type="text" placeholder="Full name">
-                 <input type="email" placeholder="Email address">
-                 <textarea>Enter text here</textarea>
-                 <button type="button">Get in touch</button>
+             <form action="https://formspree.io/f/moqrdrwj" method="POST">
+                 <input type="text" name="name" maxlength="30" placeholder="Full name" required>
+                 <input type="email" name="email" placeholder="Email address" required>
+                 <textarea name="Message" type="text" maxlength="500" placeholder="Enter text here" required></textarea>
```
Comment (Daphineatim) [inline index.html]: * Please do not use the attribute "type." It is not allowed on element "textarea" at this point.

**Review by Daphineatim — COMMENTED** [review]
Hi @sainingo ,
👍 This is good so far, your project is really nice, congratulations.

There are just a few changes you need to make for you to go to the next project.

# Required Changes ♻️ 
* Some checks were not successful, please check for linter errors and make all green checks.
 Hint, to auto fix the errors, in terminal you can run **npx hint .**
* Please read the inline comment.

*

File: index.html (hunk 1 above)
Comment (sainingo) [reply index.html]: Thanks @Daphineatim,
I am going to work on that

**Review by ReshyResh — APPROVED** [review]
# Hi @sainingo , 

### Your form works as it should and the milestone is complete!  There is nothing else to say other than... it's time to merge it!
#### Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**